labgrid/driver/networkusbstoragedriver.py

```diff
@@ -46,6 +50,53 @@ def on_activate(self):
     def on_deactivate(self):
         pass
 
+    def _pmount(self, dev_path, mount_path):
+        mount_args = ["pmount", dev_path]
+        self.logger.debug('Mount {} to {}'.format(dev_path, mount_path))
```

Comment:
Please don't use format in logging methods, rather use % style formatting and leave the formatting to logging.